### PR TITLE
Adds test for multiple claims in one claim statement

### PR DESCRIPTION
**Pull request by raulverag**

### CODE REVIEW

File: src/dataflow/analysis/tests/analysis-test.ts [hunk 1]
```diff
@@ -322,6 +322,42 @@ describe('FlowGraph validation', () => {
     assert.sameMembers(result.failures, [`'check bar is tag1 or is tag2' failed for path: P2.foo -> P3.bar`]);
   });
 
+  it(`succeeds when a check including multiple anded tags is met by a single claim`, async () => {
+    const graph = await buildFlowGraph(`
+    particle P1
```
Comment (csilvestrini) [inline src/dataflow/analysis/tests/analysis-test.ts]: Indent this whole string by 2 spaces

File: src/dataflow/analysis/tests/analysis-test.ts [hunk 2]
```diff
@@ -322,6 +322,42 @@ describe('FlowGraph validation', () => {
...
+      P2
+        bar <- h
+    `);
+    const result = validateGraph(graph);
+    assert.isTrue(result.isValid);
+  });
+
+  it(`succeeds when a check including multiple ored tags is met by a single claim`, async () => {
+    const graph = await buildFlowGraph(`
+    particle P1
```
Comment (csilvestrini) [inline src/dataflow/analysis/tests/analysis-test.ts]: Here too

File: src/dataflow/analysis/tests/analysis-test.ts (hunk 1 above)
Comment (raulverag) [reply src/dataflow/analysis/tests/analysis-test.ts]: Done.

File: src/dataflow/analysis/tests/analysis-test.ts (hunk 2 above)
Comment (raulverag) [reply src/dataflow/analysis/tests/analysis-test.ts]: Ditto.

**Pull request merged**